ci/scripts/bazel-tests.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e -x
+
+# This doesn't install dependencies, but should otherwise behave the
+# same as what CI would do on a pull request
+DEFAULT="master"
+
+# use tests from a condonedlist, if a target is moved or disabled, it should
+# also be moved in the following list in order for CI to pass.
+TESTS=(
```

Comment:
Is there a particular reason this can't be a denylist? Do you know if there's a blaze query you can write to expand a glob and delete things from it?

Reply:
It was more work, but ends up being how we then use the tags to disqualify targets.